src/main/java/graphql/testrunner/TestRunnerApplication.java

```diff
@@ -0,0 +1,45 @@
+package graphql.testrunner;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+import graphql.testrunner.service.CommandExecutorService;
+import graphql.testrunner.util.TestRunnerException;
+
+import static graphql.testrunner.service.CommandExecutorService.INSTALL_GIT;
+
+@SpringBootApplication
+public class TestRunnerApplication {
```

Comment:
I couldn't run this in intellij. It fails with:
```
java.lang.IllegalStateException: Failed to execute CommandLineRunner
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:774) ~[spring-boot-2.7.0.jar:2.7.0]
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:755) ~[spring-boot-2.7.0.jar:2.7.0]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315) ~[spring-boot-2.7.0.jar:2.7.0]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1306) ~[spring-boot-2.7.0.jar:2.7.0]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1295) ~[spring-boot-2.7.0.jar:2.7.0]
	at graphql.testrunner.TestRunnerApplication.main(TestRunnerApplication.java:25) ~[classes/:na]
Caused by: graphql.testrunner.util.TestRunnerException: null
	at graphql.testrunner.service.CommandExecutorService.writeOutput(CommandExecutorService.java:72) ~[classes/:na]
	at graphql.testrunner.service.CommandExecutorService.execute(CommandExecutorService.java:42) ~[classes/:na]
	at graphql.testrunner.service.CommandExecutorService.executeCommand(CommandExecutorService.java:29) ~[classes/:na]
	at graphql.testrunner.TestRunnerApplication.lambda$initGit$0(TestRunnerApplication.java:30) ~[classes/:na]
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:771) ~[spring-boot-2.7.0.jar:2.7.0]
	... 5 common frames omitted
```

Is this expected to work?

Reply:
Yeah, locally we will get this error. But it works fine inside the container.